modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replication/request/CommittableTxRequest.java

```diff
@@ -32,6 +32,13 @@ public interface CommittableTxRequest extends ReplicaRequest {
      */
     boolean full();
 
+    /**
+     * Get the transaction coordinator inconsistent ID.
+     *
+     * @return the transaction coordinator inconsistent ID
+     */
+    String txCoordinatorId();
```

Comment:
I'd say that it should be a field in ReadWriteReplicaRequest instead of CommittableTxRequest.